sbd.spec

```diff
@@ -97,6 +105,17 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun sbd.service
 %systemd_postun sbd_remote.service
+%else
+/sbin/chkconfig --add sbd_helper || :
```

Comment:
nope it is supposed to be a %post scriptlet but the %post is obviously missing ...